CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Bug-fixes
 
 * Fix a type error in `README.Data.Fin.Relation.Unary.Top` within the definition of `>-weakInduction`.
 
+* Remove spurious definition of `Algebra.Consequences.Propositional.sel⇒idem`,
+  which duplicated lemma from `Algebra.Consequences.Base`, instead of importing
+  from `Algebra.Consequences.Setoid`.
```

Comment:
I don't think we need this CHANGELOG entry, as from the user's perspective nothing has changed right? `Algebra.Consequences.Propositional` still exports `sel⇒idem`?